### PR TITLE
update for #2802

### PR DESCRIPTION
**Pull request by kelseiv**
minor edits and update to include remote URL endpoint option for `-config` flag

### CODE REVIEW

File: content/telegraf/v1.14/administration/configuration.md [hunk 1]
```diff
@@ -19,19 +19,20 @@ telegraf config > telegraf.conf
...
 ## Configuration file locations
 
-The location of the configuration file can be set via the `--config` command
-line flag.
+Use the `-config` flag to specify the configuration file location:
 
-When the `--config-directory` command line flag is used, files ending with
-`.conf` in the specified directory will also be included in the Telegraf
+- Filename and path, for example: `-config /etc/default/telegraf`
+- Remote URL endpoint, for example: `-config http://remote-URL-endpoint`
```
Comment (kelseiv) [inline content/telegraf/v1.14/administration/configuration.md:33]: need quotes around URL?

File: content/telegraf/v1.14/administration/configuration.md [hunk 2]
```diff
@@ -19,19 +19,20 @@ telegraf config > telegraf.conf
 ```
 
 To generate a configuration file with specific inputs and outputs, you can use the
-`--input-filter` and `--output-filter` flags:
+`-input-filter` and `-output-filter` flags:
```
Comment (danielnelson) [inline content/telegraf/v1.14/administration/configuration.md:22]: As an aside, I prefer using GNU style option format like `--input-filter` over `-input-filter`, even though Go doesn't require the extra dash.  This style is very common in Linux, and also I like to keep open the option of switching to a more powerful argument parsing library that uses the GNU style arguments and allows for short options.

File: content/telegraf/v1.14/administration/configuration.md (hunk 1 above)
Comment (danielnelson) [reply content/telegraf/v1.14/administration/configuration.md:33]: Depends on the exact URL and the shell in use, but I'd recommend showing it quoted

File: content/telegraf/v1.14/introduction/getting-started.md [hunk 3]
```diff
@@ -10,39 +10,46 @@ menu:
     parent: Introduction
 ---
 
-## Getting started with Telegraf
-Telegraf is an agent written in Go for collecting metrics and writing them into InfluxDB or other possible outputs.
-This guide will get you up and running with Telegraf.
-It walks you through the download, installation, and configuration processes, and it shows how to use Telegraf to get data into InfluxDB.
+Use Telegraf (an agent written in Go) to collect and write metrics into InfluxDB or other supported outputs.
```
Comment (danielnelson) [inline content/telegraf/v1.14/introduction/getting-started.md:13]: I'd love to come up with a better description of Telegraf.   I think mentioning Go so prominently isn't needed and doesn't make sense, maybe we should drop it.

File: content/telegraf/v1.14/administration/configuration.md (hunk 2 above)
Comment (kelseiv) [reply content/telegraf/v1.14/administration/configuration.md:22]: Thank you, @danielnelson! I appreciate the GNU standard. Would like to keep same standard across our docs, and will discuss w the docs team tomorrow.

File: content/telegraf/v1.14/introduction/getting-started.md (hunk 3 above)
Comment (kelseiv) [reply content/telegraf/v1.14/introduction/getting-started.md:13]: Tried hiding in parens, but dropping....even better!

**Pull request merged**